client/my-sites/domains/domain-management/domain-connect/domain-connect-authorize.jsx

```diff
@@ -31,8 +31,17 @@ class DomainConnectAuthorize extends Component {
 		const { providerId, serviceId, params, translate } = this.props;
 		const { domain } = params;
 
-		wpcom
-			.getDnsTemplateRecords( domain, providerId, serviceId, params )
+		wpcom.req
+			.post(
+				'/domains/' +
+					domain +
+					'/dns/providers/' +
+					providerId +
+					'/services/' +
+					serviceId +
+					'/preview',
+				{ params }
```

Comment:
Should this be:

```suggestion
				{ variables: params }
```

Because the API endpoint expects a `variables` and not a `params` property?

Reply:
Ah, very nice. I see how that destructuring was working now, thank you!